PyPoE/poe/file/bundle.py

```diff
@@ -377,6 +383,17 @@ def files(self) -> List[str]:
         return [x.rsplit('/', maxsplit=1)[-1] for x in self.paths]
 
 
+def _hash_path_3_21_2(path, seed):
+    if isinstance(path, str):
+        path = path.encode('utf-8')
+    elif not isinstance(path, bytes):
+        raise TypeError('path must be a string')
+    if path.endswith(b'/'):
+        path = path.strip(b'/')
+    path = path.lower()
+    return murmur2_64a(path, seed)
+
```

Comment:
There is potential to tighten this up here.. For instance:
```py
from pathlib import Path

...

p = Path(path) # should raise a pretty good error here automatically if not bytes or str that results in path or you can encode it as you pass.
# The explicit raise here actually obscures the error a bit I think?

...
```

Up to you if you wanna change it though..

Reply:
This code was copied and adapted verbatim from the existing logic as I was not certain in what contexts it was called, I left them alone as they are internal functions.
It could probably be cleaned up, but as #115 has referenced some of these commits now I'm not sure how much of a mess that additional work would cause.

Reply:
We can defer changes to after the  merge then